test/e2e/edu-5490-helpers.js

```diff
@@ -0,0 +1,150 @@
+        .clickIf('input[name="root_previousBenefits_chapter35"]', previousBenefits.chapter35)
+        .clickIf('input[name="root_previousBenefits_chapter33"]', previousBenefits.chapter33)
+        .clickIf('input[name="root_previousBenefits_transferOfEntitlement"]', previousBenefits.transferOfEntitlement)
+        .fill('input[name="root_previousBenefits_veteranFullName_first"]', previousBenefits.veteranFullName.first)
+        .fill('input[name="root_previousBenefits_veteranFullName_middle"]', previousBenefits.veteranFullName.middle)
+        .fill('input[name="root_previousBenefits_veteranFullName_last"]', previousBenefits.veteranFullName.last)
+        .selectDropdown('root_previousBenefits_veteranFullName_suffix', previousBenefits.veteranFullName.suffix)
+        .fill('input[name="root_previousBenefits_veteranSocialSecurityNumber"]', previousBenefits.veteranSocialSecurityNumber);
+    }
+
+    client.fill('input[name="root_previousBenefits_other"]', previousBenefits.other);
+  }
+}
+
+function completeSponsorService(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const activeDuty = `root_currentlyActiveDuty${data.currentlyActiveDuty ? 'Yes' : 'No'}`;
+    const felony = `root_outstandingFelony${data.outstandingFelony ? 'Yes' : 'No'}`;
+    client
+      .fill('input[name="root_serviceBranch"]', data.serviceBranch)
+      .click(`input[name="${activeDuty}"]`)
+      .click(`input[name="${felony}"]`);
+  }
+}
+
+function completeSecondaryContact(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const { fullName, phone, address } = data.secondaryContact;
+    client
+      .fill('input[name="root_secondaryContact_fullName"]', fullName)
+      .fill('input[name="root_secondaryContact_phone"]', phone)
+      .selectDropdown('root_secondaryContact_address_country', address.country)
+      .fill('input[name="root_secondaryContact_address_street"]', address.street)
+      .fill('input[name="root_secondaryContact_address_street2"]', address.street2)
+      .fill('input[name="root_secondaryContact_address_city"]', address.city)
+      .selectDropdown('root_secondaryContact_address_state', address.state)
+      .fill('input[name="root_secondaryContact_address_postalCode"]', address.postalCode);
+  }
+}
+
+function completeEducationHistory(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const completionDate = _.get('highSchool.highSchoolOrGedCompletionDate', data);
+
+    client.selectDropdown('root_highSchool_status', data.highSchool.status);
+    if (completionDate) {
+      client.fillDate('root_highSchool_view:highSchoolOrGedCompletionDate', completionDate);
+    }
+
+    if (!_.isEmpty(data.postHighSchoolTrainings)) {
+      // Open up the trainings section if there are trainings in the data
+      client
+        .click('input[name="root_view:hasTrainingsYes"]')
+        .waitForElementVisible('input[name="root_postHighSchoolTrainings_0_name"]', Timeouts.normal);
```

Comment:
The expandable fields seem to have this problem, but I don't know that it's consistent...Perhaps they don't need this sometimes because in some places nightwatch fills out other information while this is expanding?